members/manager.py

```diff
@@ -32,3 +32,11 @@ async def ban(ctx: discord, member: discord.Member):
         )
         await ctx.send("{0.name} foi removido por inatividade".format(member))
         await ctx.guild.kick(member)
+
+
+@commands.command()
+async def stonks(ctx: discord, code: str):
```

Comment:
Please move this command to the package for stonks. I suggest to create a module named: `/stonks/commands.py`